src/tremotesf/ui/screens/addtorrent/addtorrentdialog.cpp

```diff
@@ -233,7 +236,11 @@ namespace tremotesf
         const auto updateUi = [=] {
             const bool enabled = mRpc->isConnected() && (mFilesModel ? mFilesModel->isLoaded() : true);
             if (enabled) {
-                mDownloadDirectoryWidget->updateComboBox(mRpc->serverSettings()->downloadDirectory());
+                if (Settings::instance()->rememberDownloadDir() && Settings::instance()->lastDownloadDirectory() != "") {
```

Comment:
When QString is compared to string literal, new temporary QString object is created from string literal for the purpose of comparison. While in this case it's likely cheap because QString has empty string optimization, calling isEmpty() would be better solution.

Reply:
I'll fix this. Thanks for the review.